content/docs/command-reference/exp/run.md

```diff
@@ -63,14 +71,21 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 > In addition to the following, `dvc exp run` accepts the options in `dvc repro`
 > except for `--glob`, `--no-commit`, and `--no-run-cache`.
 
-- `-S [<filename>:]<param_name>=<param_value>`,
-  `--set-param [<filename>:]<param_name>=<param_value>` - set the value of
-  existing `dvc params` for this experiment. `filename` can be any valid params
-  file (`params.yaml` by default). This will override the param values coming
-  from the params file.
+- `-S [<filename>:]<override_pattern>`,
+  `--set-param [<filename>:]<override_pattern>` - set the value of existing
+  `dvc params` for this experiment. `filename` can be any valid params file
+  (`params.yaml` by default). This will override the param file before running
+  the experiment.
```

Comment:
"override" could be easily misread as "overwrite" here. I think the previous wording was clearer, or did the behavior change?